.azure-pipelines/config.yml

```diff
@@ -36,7 +36,7 @@ jobs:
   - script: |
       set -euo pipefail
 
-      lintver="1.23.8"
+      lintver="1.42.0"
```

Comment:
If you change the lint version you will have to fix all of these: https://dev.azure.com/signalfx/signalfx-agent/_build/results?buildId=6029&view=logs&jobId=18a12b53-e25b-5b79-411d-9a29f3a83d7c&j=18a12b53-e25b-5b79-411d-9a29f3a83d7c&t=becae859-8d01-508d-6d9d-600a82ae380c